.github/workflows/avm.platform.sync-avm-modules-list.yml

```diff
@@ -0,0 +1,36 @@
+name: "avm.platform.sync-avm-modules-list"
+
+on:
+  schedule:
+    - cron: "30 4 * * *" # Every day at 4:30 am
+  workflow_dispatch:
+
+jobs:
+  sync-avm-modules-list:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: tibdex/github-app-token@v2
```

Comment:
Are we good with leveraging external actions in this repo?

Reply:
@eriqua yes. We discussed with Alex already. This action is already used by Jack and Chris.

Reply:
Resolving as already addressed